ops/packer/scripts/platinum/03-install-security-updates.sh

```diff
@@ -2,3 +2,6 @@
 
 # Apply security patches
 sudo yum update-minimal --security -y
+
+# Aggressively reconfigure grub configuration
+sudo grub2-mkconfig -o /boot/grub2/grub.cfg
```

Comment:
We might consider leaving this out of #2493 because it is strictly orthogonal. However, as implemented, this works.

Reply:
Honestly, I'm perfectly OK including it here. In my opinion, it's not worth creating a new ticket for 1 line, and we want to get our Platinums working again ASAP anyways.